lib/streamlit/hashing.py

```diff
@@ -230,11 +230,29 @@ def is_simple(obj):
     return NoResult
 
 
+def _get_fqn_of_type(the_type):
+    """Get module.type_name for a given type."""
+    module = the_type.__module__
+    name = the_type.__qualname__
+    return "%s.%s" % (module, name)
```

Comment:
There's some code duplication between this and `type_util.get_fqn_of_type()`. Also the name of this function is confusingly similar to the `type_util` one.

Why not move this function into `util` and rename it `get_fqn()`, then use it inside `type_util.get_fqn_of_type()`?